content/pt-br/docs/tasks/administer-cluster/kubelet-credential-provider.md

```diff
@@ -0,0 +1,147 @@
+
+Você pode estar interessado em usar essa funcionalidade se alguma das condições abaixo for verdadeira:
+
+* Chamadas de API para um serviço de provedor de nuvem são necessárias para recuperar informações de autenticação para um registro.
+* As credenciais têm tempos de expiração curtos e é necessário solicitar novas credenciais com frequência.
+* Armazenar credenciais de registro no disco ou em `imagePullSecrets` não é aceitável.
+
+Este guia demonstra como configurar o mecanismo de plugin do provedor de credenciais de imagem do kubelet.
+
+## {{% heading "prerequisites" %}}
+
+* Você precisa de um cluster Kubernetes com nós que suportem plugins de provedor de credenciais do kubelet. Esse suporte está disponível no Kubernetes {{< skew currentVersion >}}; As versões v1.24 e v1.25 do Kubernetes incluíram isso como um recurso beta, ativado por padrão.
+
+* Uma implementação funcional de um plugin executável de provedor de credenciais. Você pode criar seu próprio plugin ou usar um fornecido por provedores de nuvem.
+
+{{< version-check >}}
+
+<!-- steps -->
+
+## Instalando Plugins nos Nós
+
+Um plugin de provedor de credenciais é um binário executável que será executado pelo kubelet. Certifique-se de que o binário do plugin exista em cada nó do seu cluster e esteja armazenado em um diretório conhecido. O diretório será necessário posteriormente ao configurar as _flags_ do kubelet.
+
+## Configurando o Kubelet
+
+Para usar esse recurso, o kubelet espera que duas flags sejam definidas:
+
+* `--image-credential-provider-config` - o caminho para o arquivo de configuração do plugin de provedor de credenciais.
+* `--image-credential-provider-bin-dir` - o caminho para o diretório onde estão localizados os binários do plugin de provedor de credenciais.
+
+### Configurar um provedor de credenciais do kubelet
+
+O arquivo de configuração passado para `--image-credential-provider-config` é lido pelo kubelet para determinar quais plugins executáveis devem ser invocados para quais imagens de contêiner. Aqui está um exemplo de arquivo de configuração que você pode acabar usando se estiver usando o plugin baseado no [ECR](https://aws.amazon.com/ecr/):
+
+```yaml
+apiVersion: kubelet.config.k8s.io/v1
+kind: CredentialProviderConfig
+# providers é uma lista de plug-ins auxiliares do provedor de credenciais que serão habilitados pelo kubelet.
+# Vários provedores podem corresponder a uma única imagem, caso em que as credenciais
+# de todos os provedores serão devolvidos ao kubelet. Se vários provedores forem chamados
+# para uma única imagem, os resultados são combinados. Se os provedores retornarem 
+# chaves de autenticação sobrepostas, o valor do provedor anterior da lista é usado.
+providers:
+   # name é o nome necessário do provedor de credenciais. Deve corresponder ao nome do
+   # executável do provedor visto pelo kubelet. O executável deve estar no 
+   # diretório bin do kubelet (definido pelo sinalizador --image-credential-provider-bin-dir).
```

Comment:
```suggestion
   # diretório bin do kubelet (definido pela flag --image-credential-provider-bin-dir).
```
Very minor, mainly for consistency because you used flag before.